index.html

```diff
@@ -244,14 +252,15 @@ <h2>
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
             </h2>
         </section>
-        <form class="connect" action="https://formspree.io/f/mwkjdnnq" method="POST" autocomplete="off">
+        <form class="connect" action="https://formspree.io/f/mzblpang" method="POST" autocomplete="off" id="myForm">
```

Comment:
- [ ] After submitting the form, it doesn't return the confirmation message. Please 🙏🏽 fig this issue for a better UX.